packages/connect-explorer/src/components/Method.tsx

```diff
@@ -225,6 +225,23 @@ export const VerifyButton = ({ name, onClick }: VerifyButtonProps) => {
     return <Button onClick={() => onClick(verifyUrls[index])}>Verify response</Button>;
 };
 
+type SubmitButtonProps = Omit<ButtonProps, 'children'> & {
+    text?: string;
+};
+
+const SubmitButton = ({ onClick, text, isFullWidth, isLoading }: SubmitButtonProps) => {
```

Comment:
You are using only 3 props from `ButtonProps` here. So I can use for example 
`<SubmitButton variant="tertiary" ... />` and I don't get any type error even though it's not working properly. 

What about specify type like this?

```tsx
type SubmitButtonProps = {
    onClick: ButtonProps["onClick"];
    isFullWidth: ButtonProps["isFullWidth"];
    isLoading: ButtonProps["isLoading"];
    text?: string;
};

```

Reply:
sounds right, fixed up